microcontroller/include/debug.h

```diff
@@ -0,0 +1,10 @@
+#ifndef DEBUG_H
+#define DEBUG_H
+
+#include "SEGGER_RTT.h"
+
+#define DEBUG 1
```

Comment:
Wouldn't it be better to add this line to an amperose_conf.h file where all configurations are regrouped ?

Reply:
I don't mind, I'll move it